extra/xkcd/xkcd.factor

```diff
@@ -13,8 +13,8 @@ IN: xkcd
 
 : comic-image ( url -- image )
     http-get nip
-    R/ http:\\/\\/imgs\.xkcd\.com\\/comics\\/[^\.]+\.(png|jpg)/
-    first-match >string load-http-image ;
+    R/ \/\/imgs\.xkcd\.com\\/comics\\/[^\.]+\.(png|jpg)/
+    first-match >string "http:" prepend load-http-image ;
```

Comment:
?

Reply:
It broke. This should go into master/ anyway, oops